client/components/SideNavBar.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router';
+import jwt from 'jsonwebtoken';
+
+class SideNavBar extends React.Component {
+  render() {
+    // conditional rendering for admin to access roles link
+    const curUser = jwt.decode(localStorage.jwtToken);
+    let rolesLink = null;
+    if (curUser) {
+      const curUserRole = curUser.roleId;
+      if (curUserRole === 1) {
+        rolesLink = (
+          <li>
+            <Link to="/roles" className="waves-effect" id="roles">
+              <i className="material-icons">domain</i>Roles
+            </Link>
+          </li>
+        );
+      }
+    }
+    return (
+      <div>
+        <ul id="slide-out" className="side-nav">
+          <li>
+            <div className="background">
+              <img src={require('../images/docs1.jpg')} width="300px"/>
```

Comment:
This is not the best way to import images. Check how to import and image the same way you import files. Correct all other occurrences of this.